includes/polling/applications/zfs.inc.php

```diff
@@ -2,16 +2,17 @@
 
 use LibreNMS\RRD\RrdDefinition;
 
-echo ' zfs ';
-
 $name = 'zfs';
 $app_id = $app['app_id'];
-$options = '-O qv';
-$mib = 'NET-SNMP-EXTEND-MIB';
-$oid = 'nsExtendOutputFull.3.122.102.115';
-$json = snmp_get($device, $oid, $options, $mib);
 
-$zfs=json_decode(stripslashes($json), true);
+echo $name;
+
+$zfs=json_app_get($device, '3.122.102.115');
```

Comment:
I think it would better to just use a string as the second argument.

Let me just drop this here:
```
function str2index($string)
{
    $ascii = array_map('ord', str_split($string));
    array_unshift($ascii, strlen($string));
    return implode('.', $ascii);
}
```

Reply:
I am not sure what you are trying to accomplish with this.

Reply:
Ahh! Okay! I see what you mean now. By string you mean "zfs" instead of '3.122.102.115'.